Orange/tests/test_dasktable.py

```diff
@@ -74,7 +74,7 @@ def _wrapper(self):
         test_case(self, lambda table: table)
         self.tearDown()
         self.setUp()
-        test_case(self, lambda table: temp_dasktable(table))
```

Comment:
Why did you need to change this? (I forgot why there were lambdas here)

Reply:
It seemed like the lambda doesn't need to be there, temp_dasktable is already callable.
I can't remember if there was a reason to write it like that in the first place either.

Reply:
Ah, I see, forget it.